docker/docker-compose.indexer-test.yml

```diff
@@ -0,0 +1,116 @@
+services:
+  # 1. Storage Service - starts first
+  linera-storage-service:
+    image: "${LINERA_INDEXER_IMAGE:-linera-indexer-test}"
+    container_name: linera-storage-service
+    ports:
+      - "${LINERA_STORAGE_SERVICE_PORT:-1235}:${LINERA_STORAGE_SERVICE_PORT:-1235}"
+    command: >
+      sh -c "./linera-storage-server memory --endpoint 0.0.0.0:${LINERA_STORAGE_SERVICE_PORT:-1235}"
+    environment:
+      - LINERA_STORAGE_SERVICE_PORT=${LINERA_STORAGE_SERVICE_PORT:-1235}
+    healthcheck:
+      test: ["CMD-SHELL", "nc -z localhost ${LINERA_STORAGE_SERVICE_PORT:-1235}"]
+      interval: 5s
+      timeout: 3s
+      retries: 10
+      start_period: 10s
+    networks:
+      - linera-network
+
+  # 2. Indexer - starts after storage service is healthy
+  linera-indexer:
+    image: "${LINERA_INDEXER_IMAGE:-linera-indexer-test}"
+    container_name: linera-indexer
+    ports:
+      - "${INDEXER_PORT:-8081}:${INDEXER_PORT:-8081}"
+    # working_dir: /linera-indexer/example
+    command: >
+      sh -c "set -ex; touch ${INDEXER_DATABASE_PATH:-/data/indexer.db} &&./linera-indexer-grpc --port ${INDEXER_PORT:-8081} --database-path ${INDEXER_DATABASE_PATH:-/data/indexer.db}"
```

Comment:
TODO: touch shouldn't be necessary. The SQLite constructor should create the file. Update the indexer code to do it.